bears/uploadTool.py

```diff
@@ -0,0 +1,68 @@
+from Constants import classifiers, imports, encoding, requirements
+from Constants import setup_constants
+from coalib.bears.requirements.PythonRequirement import PythonRequirement
+import shutil
+import os
+import glob
+import re
+
+if not os.path.exists('upload/'):
+    os.mkdir('upload/')
+
+#Reqrs = MockBear.REQUIREMENTS
+
+# for req in Reqrs:
+#    if type(req) == PythonRequirement:
+#        if req.version:
+#            reqs.write(req.package + '==' + req.version + '\n')
+#        else:
+#            reqs.write(req.package + '\n')
+
+bears = glob.glob('**/*Bear.py')
+
+for bear in bears:
+    file = bear
+    bear = re.match(r'.+/(.*)\.py', bear).group(1)
+    if not os.path.exists('upload/' + bear):
+        os.mkdir('upload/' + bear)
+        if not os.path.exists('upload' + bear + '/' + bear):
+            os.mkdir('upload/' + bear + '/' + bear)
+    if not os.path.exists('upload/' + bear + '/' + bear + '/__init__.py'):
+        init = open('upload/' + bear + '/' + bear + '/__init__.py', 'w')
+        init.write(' ')
+        init.close()
+    shutil.copyfile(file, 'upload/' + bear + '/' + bear + '/' + bear + '.py')
+
+    setup = open('upload/' + bear + '/setup.py', "w")
```

Comment:
This looks like a great place to make use of `with open(...) as fd:` to save yourself the `.close()` in line 61